code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -94,9 +85,118 @@
+			continue
+		reachable_turfs += adjacent_turf
+
+	var/peels_to_spawn = min(peel_amount, reachable_turfs.len)
+	for(var/i in 1 to peels_to_spawn)
+		new banana_type(pick_n_take(reachable_turfs))
+	playsound(owner, 'sound/creatures/clown/clownana_rustle.ogg', 100)
+	animate(owner, time =1, pixel_x = 6, easing = CUBIC_EASING | EASE_OUT)
+	animate(time = 2, pixel_x = -8, easing = CUBIC_EASING)
+	animate(time =  1, pixel_x = 0, easing = CUBIC_EASING | EASE_IN)
+	StartCooldown()
+
+///spawns a plumb bunch of bananas imbued with mystical power.
+/datum/action/cooldown/exquisite_bunch
+	name = "Exquisite Bunch"
+	desc = "Pluck your finest bunch of bananas from your head. This bunch is especially nutrious to monkeykind. A gentle tap will trigger an explosive ripening process."
+	icon_icon = 'icons/obj/hydroponics/harvest.dmi'
+	cooldown_time = 60 SECONDS
+	button_icon_state = "banana_bunch"
+	background_icon_state = "bg_nature"
+	///If we are currently activating our ability.
+	var/activating = FALSE
+
+/datum/action/cooldown/exquisite_bunch/Trigger(atom/target)
+	if(activating)
+		return
+	var/bunch_turf = get_step(owner.loc, owner.dir)
+	if(!bunch_turf)
+		return
+	if(!owner.CanReach(bunch_turf) || !isopenturf(bunch_turf))
+		owner.balloon_alert("can't do that here!")
+		return
+	activating = TRUE
+	if(!do_after(owner, 1 SECONDS))
+		activating = FALSE
+		return
+	playsound(owner, 'sound/creatures/clown/hehe.ogg', 100)
+	if(!do_after(owner, 1 SECONDS))
+		activating = FALSE
+		return
+	activating = FALSE
+	return ..()
+
+/datum/action/cooldown/exquisite_bunch/Activate(atom/target)
+	. = ..()
+	new /obj/item/food/grown/banana/bunch(get_step(owner.loc, owner.dir))
+	playsound(owner, 'sound/items/bikehorn.ogg', 60)
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/playsound, owner, 'sound/creatures/clown/hohoho.ogg', 100, 1), 1 SECONDS)
+	StartCooldown()
+
+/obj/item/food/grown/banana/bunch
+	name = "banana bunch"
+	desc = "Am exquisite bunch of bananas. The almost otherwordly plumpness steers the mind any discening entertainer towards the divine."
+	icon_state = "banana_bunch"
+	bite_consumption_mod = 4
+	var/is_ripening = FALSE
+
+/obj/item/food/grown/banana/bunch/Initialize(mapload, obj/item/seeds/new_seed)
+	. = ..()
+	reagents.add_reagent(/datum/reagent/consumable/monkey_energy, 10)
+	reagents.add_reagent(/datum/reagent/consumable/banana, 10)
+
+/obj/item/food/grown/banana/bunch/proc/start_ripening()
+	if(is_ripening)
+		return
+	playsound(src, 'sound/effects/fuse.ogg', 80)
+
+	animate(src, time = 1, pixel_z = 12, easing = ELASTIC_EASING)
+	animate(time = 1, pixel_z = 0, easing = BOUNCE_EASING)
+	addtimer(CALLBACK(src, .proc/explosive_ripening), 3 SECONDS)
+	for(var/i in 1 to 32)
+		animate(color = (i % 2) ? "#ffffff": "#ff6739", time = 1, easing = QUAD_EASING)
+
+/obj/item/food/grown/banana/bunch/proc/explosive_ripening()
+	honkerblast(src, light_range = 3, medium_range = 1)
```

Comment:
you have to wait 3 seconds for an area of effect this small?

Reply:
yes